spec/imsc-hrm.html

```diff
@@ -237,9 +237,154 @@ <h2>Terms and Definitions</h2>
+      [[IMSC]] by imposing constraints on the contents of the presentation.</p>
+  
+      <p>Because of the temporal and spatial variability of subtitles and
+        captions across types of content, territories and languages, it is not
+        possible to limit the complexity of an <a>IMSC Document Instance</a> using
+        only average values.</p>
+  
+      <p class="example">A constraint of 840 characters per minute would
+        allow a subtitle to contain 840 characters, as long it was the only one
+        within a 60 second window.</p>
+  
+      <p>The HRM achieves a more accurate representation of the complexity of an
+        <a>IMSC Document Instance</a> at any given time by taking into account
+        its past complexity in addition to its instantaneous complexity. The
+        same approach is commonly used in video to limit bitstream complexity,
+        e.g., the Hypothetical Reference Decoder (HRD) specified in
+        [[iso14496-10]].
+      </p>
+    </section>
+  
+    <section>
+      <h3>How does the HRM measures complexity?</h3>
+  
+      <p>
+        The HRM defines an (hypothetical) time required for painting subtitles
+        and captions. Painting includes drawing region backgrounds, rendering
+        and copying text, and decoding and copying images. Complexity is then
+        limited by requiring that the time to paint a subtitle/caption is
+        shorter than the time elapsed since the previous subtitle/caption.
+      </p>
+  
+      <p>The calculation of the time required for painting subtitles and
+      captions is based on a static analysis of the <a>IMSC Document
+      Instances</a> and requires no fetching of external resources or actual
+      rendering. It however assumes that some characters are faster to render
+      than others, based on whether they were previously rendered and on the
+      kind of glyph they generate.<p>
+        
+      <p>The HRM is not intended as an accurate calculation of rendering time,
+      but is intended to scale with actual rendering times: a document that is
+      twice as complex according to the HRM would require roughly twice as many
+      resources to actually render.
+      </p>
+    </section>
+  
+    <section>
+      <h3>Where is the HRM used?</h3>
+  
+      <p>The HRM is typically used prior to distribution of the <a>IMSC Document
+      Instance</a> to end-user, as an integral part of authoring and as a
+      quality check before distribution.</p>
+  
+      <p>The consequences of an <a>IMSC Document Instance</a> exceeding the HRM
+      limits will depend on the circumstances:</p>
+  
+      <ul>
+        <li>an authoring system might, for example, flag the specific point in
+        time where the HRM limts were exceeded;</li>
+        <li>the ingest component of a streaming platform might outright reject
+        the <a>IMSC Document Instance</a>.</li>
+      </ul>
```

Comment:
Suggest including presentation processor here too:

```suggestion
      <li>a player might be unable to present the subtitles and captions correctly,
      for example, it might not present some subtitles or captions at all,
      or it might present them too late.</li>
      </ul>
```